service/authService.ts

```diff
@@ -0,0 +1,26 @@
+import { Login } from "../model/login";
+const axios = require("axios");
+
+module.exports.login = async function(login: Login): Promise<string> {
+    try {
+        const response = await axios.post("http://localhost:8080/api/login",login)
+
+        return response.data
+    } catch(e) {
+        throw new Error("Could not login");
+    }
+}
+
+
+    module.exports.register = async function(login: Login): Promise<boolean> {
```

Comment:
Would it not make more sense for an object of type register to be passed in here?

Reply:
No i don't think so, I think it would be computationally more performant if the register object is converted to the login in the controller